pkg/hub/lease/controller.go

```diff
@@ -26,6 +26,7 @@ import (
 
 const leaseDurationTimes = 5
 const leaseName = "managed-cluster-lease"
+const clusterNameLabel = "open-cluster-management.io/cluster-name"
```

Comment:
we should put all common lables/annotations in the api repo with good documents. Please create an issue in api repo.

Reply:
created an issue for this https://github.com/open-cluster-management-io/api/issues/194